docs/guide/dev/configuration/file_browse_upload/ckfinder_integration/README.md

```diff
@@ -1,10 +1,16 @@
+---
+category: file-manager
+order: 60
+url: guide/dev_ckfinder_integration
+menu-title: CKFinder Integration
+meta-title-short: CKFinder Integration
+---
 # CKFinder Integration
 
 The aim of this article is to explain how to integrate CKEditor with [CKFinder](http://cksource.com/ckfinder/), a powerful and easy to use Ajax file manager. See [the demo](http://cksource.com/ckfinder/demo#ckeditor) for a live demonstration.
 
-<p class="requirements">
-	CKFinder is a <a href="http://cksource.com/ckfinder">commercial application</a> that was designed with CKEditor compatibility in mind. It is currently available as version 3.x for PHP and version 2.x for Java, ASP.NET, ASP and ColdFusion.
-</p>
+<info-box info="">
 CKFinder is a <a href="http://cksource.com/ckfinder">commercial application</a> that was designed with CKEditor compatibility in mind. It is currently available as version 3.x for PHP and version 2.x for Java, ASP.NET, ASP and ColdFusion.

```

Comment:
What is a `^M` character? 🙂 

![image](https://user-images.githubusercontent.com/5353898/35110792-f5cede02-fc79-11e7-8332-0d926d9b914a.png)

Reply:
It appeared instead of a new line character I think. I don't know why